### ISSUE TITLE
Typescript - forced dependency to koa/express

### ISSUE THREAD

**Issue by PiTiLeZarD** [flash-oss/graphql-upload-minimal#23]
Hi I'm using typescript and express and when I try to use this library, I get a:

```
node_modules/graphql-upload-minimal/index.d.ts:4:58 - error TS2307: Cannot find module 'koa' or its corresponding type declarations.

4 import { DefaultContext, DefaultState, Middleware } from "koa";
```

And I don't really want to install koa on the side just for this. I'll try to trick this somehow but it'd be nice to fix it ;)

**Comment by koresar** [comment]
Wow. That's a sad bug.
Could you please give us a pull request since you're on it already?

**Comment by PiTiLeZarD** [comment]
I had a quick look, and I actually don't know how to fix this at the moment... I kinda feel like dynamic imports are the way to go but you can't have implementation in types declarations. Interesting problem tho, I might have another look this weekend.

**Comment by koresar** [comment]
This might help: https://www.npmjs.com/package/@types/koa

**Comment by PiTiLeZarD** [comment]
I was looking at another package (express-ipfilter), they have a similar use case and they deal with it this way:
```
```

**Comment by koresar** [comment]
I simply removed typings for those two functions. Published as v1.5.1.
Please, check if it works for you.
Thanks.
